gradle.properties

```diff
@@ -2,24 +2,50 @@
 # Copyright © 2014-2020 The Android Password Store Authors. All Rights Reserved.
 # SPDX-License-Identifier: GPL-3.0-only
 #
+org.gradle.parallel=true
+org.gradle.configureondemand=true
+org.gradle.caching=true
 
-# AndroidX
-android.enableJetifier=false
-android.useAndroidX=true
+# Enabling filesystem watching
+org.gradle.vfs.watch=true
+org.gradle.unsafe.watch-fs=true
 
-# R8 Full mode
-android.enableR8.fullMode=true
+# Enable Kotlin incremental compilation
+kotlin.incremental=true
 
-# Disable warnings for experimental options
-android.suppressUnsupportedOptionWarnings=android.suppressUnsupportedOptionWarnings,android.enableR8.fullMode,android.useMinimalKeepRules,android.namespacedRClass
+# Enable parallel tasks execution for Kotlin Gradle plugin
+kotlin.parallel.tasks.in.project=true
 
-# https://jakewharton.com/increased-accuracy-of-aapt2-keep-rules/
-android.useMinimalKeepRules=true
+# Kotlin code style
+kotlin.code.style=official
+
+# Enable incremental annotation processor for KAPT
+kapt.incremental.apt=true
+
+# Turn off AP discovery in compile path to enable compile avoidance
+kapt.include.compile.classpath=false
```

Comment:
Not currently using `kapt` but there might be some possibility in the future, so better safe than sorry.